plat_all-unix-style.go

```diff
@@ -136,11 +135,12 @@ func RenameCrossDevice(oldpath, newpath string) error {
 	return os.Rename(oldpath, newpath)
 }
 
-func (task *TaskRun) addGroupsToUser(groups []string) error {
-	if len(groups) == 0 {
-		return nil
-	}
-	return fmt.Errorf("Not able to add groups %v to user on platform %v - feature not supported", groups, runtime.GOOS)
+func (task *TaskRun) removeUserFromGroups(groups []string) (updatedGroups []string, notUpdatedGroups []string) {
+	return updatedGroups, groups
```

Comment:
is it worthwhile to add a logging call to mention that this call (and one to addUserToGroups) when the updatedGroups aren't equivalent to groups?

Reply:
That is logged in the calling methods, e.g. [here](https://github.com/taskcluster/generic-worker/blob/592fd32c28d16a1fd68a4a426a4b967f79a91359/os_groups.go#L64-L68).

Reply:
great!